### ISSUE TITLE
On welcome page, Provide a link to redirect user to create an organisation.

### ISSUE THREAD

**Issue by h2o-Noob** [zulip/zulip-mobile#5487]
On welcome page where user is asked to enter a zulip organisation url. 

<img width="341" alt="Screenshot 2022-09-14 at 11 45 44 AM" src="https://user-images.githubusercontent.com/85777684/190074964-b85074dd-f1d5-4bf7-a25f-60688fc32749.png">

If the user does'nt have any existing zulip organisation, He should be provided with a link to redirect him/her to zulip website where he/she can create an organisation.

**Comment by h2o-Noob** [comment]
working on this issue.

**Comment by chrisbobbe** [comment]
Good idea. 🙂

Perhaps we should have two links: one for people who want to make a new Zulip Cloud org, and one for people who want to set up a self-hosted org.

Or maybe there's a single page that presents both options, perhaps with some guidance for people who don't know which to choose.

@alya, thoughts? 🙂

**Comment by JoeyPriceless** [comment]
@chrisbobbe So this is how the view looks on desktop (and I'm assuming web). Perhaps the mobile version should mirror this? 

I agree that maybe there should be some guidance regarding cloud hosting and self hosting but it seems unlikely that someone who's interested in self hosting would try to setup their org through the mobile app.

![image](https://user-images.githubusercontent.com/8920712/190599016-c0ad1401-146f-4b54-a346-616a334c9315.png)

**Comment by alya** [comment]
Yeah, I think something similar to the Desktop app (styled appropriately for the mobile app context) would be fine. So some version of:

--- or ----
[Create a new organization]

I agree that  folks are unlikely to start setting up self-hosting by installing the mobile app. :)

**Comment by Syed-Ansar** [comment]
Hey @alya I am interested to take up this issue and I wanted to claim it. Can you assign me this issue.

**Comment by alya** [comment]
@Syed-Ansar please post a comment describing your proposed approach when you're ready. I can assign the issue to you once you [have a rough plan](https://zulip.readthedocs.io/en/latest/overview/contributing.html#picking-an-issue-to-work-on). Thanks!

**Comment by Syed-Ansar** [comment]
Hey @alya I have 2 approaches

1. We can create a button and redirect to web app.
2. Create a new screen for the in app registration by verifying the E-mail and creating new Organisation.

I would be very happy to work on any of the above proposal. Thank you.
